lutris/gui/application.py

```diff
@@ -77,23 +78,21 @@ def __init__(self):
         # established; this will apply to all connections from this point forward.
         init_exception_backstops()
 
-        GObject.add_emission_hook(Game, "game-start", self.on_game_start)
-        GObject.add_emission_hook(Game, "game-stopped", self.on_game_stopped)
+        GObject.add_emission_hook(GameLauncher, "game-start", self.on_game_start)
+        GObject.add_emission_hook(GameLauncher, "game-stopped", self.on_game_stopped)
         GObject.add_emission_hook(PreferencesDialog, "settings-changed", self.on_settings_changed)
 
         GLib.set_application_name(_("Lutris"))
         self.force_updates = False
         self.css_provider = Gtk.CssProvider.new()
-        self.window = None
+        self.window: LutrisWindow = None
+        self.style_manager: StyleManager = None
         self.launch_ui_delegate = LaunchUIDelegate()
         self.install_ui_delegate = InstallUIDelegate()
-
-        self._running_games = []
+        self._running_games: List[Game] = []
```

Comment:
This should be GameLaunchers, not games.

We should not create GameLaunchers on demand, we should create them on launch, and *only* on launch. Games, on the other hand, we load from the DB whenever we feel like it- a key difference.

This change would make it safe for GameLauncher to track the running game; right now we have to use _running_games to have special blessed Game objects for that purpose, and that's been a source of bugs. Let's fix that.

I might argue that _running_games might be better moved out of application.py to GameLauncher's module instead. I see no reason to tie this to the UI, and if you really do want to go to QT, you'll want to separate this from GTK.